models/form.go

```diff
@@ -12,6 +12,7 @@ type Form struct {
 	RequireLogin   bool   `json:"require_login"`
 	CollectEmail   bool   `json:"collect_email"`
 	SingleResponse bool   `json:"single_response"`
+	ResponseToken  string `json:"token"`
```

Comment:
Use `json:"-"` for `ResponseToken` instead, so it won't be returned with the form. Right now, anyone will be able to see this and view responses.

Reply:
Well not really, since we're also checking `creator`, but this will be the case after that check is removed.